services/shhext/chat/x3dh_test.go

```diff
@@ -174,7 +174,7 @@ func TestExtractIdentity(t *testing.T) {
 	)
 }
 
-// Alice wants to send a message to Bob
+// TestX3dhActive; Alice wants to send a message to Bob
```

Comment:
```suggestion
// TestX3DHActive; Alice wants to send a message to Bob
```

Reply:
Also, I think it could sound better if it's a sentence: `TestX3DHActive checks a case when Alice wants to send a message to Bob.`